src/object/cli_obj.c

```diff
@@ -617,6 +617,10 @@ obj_shards_2_fwtgts(struct dc_object *obj, uint32_t map_ver, uint64_t tgt_set,
 			if (rc != 0)
 				return rc;
 		}
+
+		if (!forward)
```

Comment:
hmm, in the case the grp_size == 1, then ort_srv_disp is false, then the logic here looks incorrect. Seems should move this to above L619.
I'm changing this code for other patch, so may need not refresh this patch if only for this, I can change it in my patch.

Reply:
So it should be:

"if (!forward && grp_size > 1) continue;"